libs/explorers/comparison-tools/src/lib/base-comparison-tool/base-comparison-tool.component.ts

```diff
@@ -26,7 +26,7 @@ export class BaseComparisonToolComponent {
   isLoading = input(true);
 
   currentConfig = this.comparisonToolService.currentConfig;
-  resultsCount = this.comparisonToolService.totalResultsCount;
+  resultsCount = this.comparisonToolService.loadingResultsCount;
```

Comment:
What do you think about changing the variable name in the component to match the service for clarity? 

```suggestion
  loadingResultsCount = this.comparisonToolService.loadingResultsCount;
```

Reply:
Sure.  In the future, it might make sense to consider it renaming it to loadingMessage.